pkg/gvks/supportedgvks/mapping.go

```diff
@@ -0,0 +1,4326 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by "scripts/generate-gvk-mapping"; DO NOT EDIT.
+
+package supportedgvks
+
+import "k8s.io/apimachinery/pkg/runtime/schema"
+
+type GVKMetadata struct {
+	Labels map[string]string
+}
+
+var SupportedGVKs = map[schema.GroupVersionKind]GVKMetadata{
```

Comment:
Could we add `generated` either in the file name or in the file path?

Reply:
Yeah, that makes sense. Done